store/tikv/coprocessor.go

```diff
@@ -308,7 +308,7 @@ type copIterator struct {
 	errChan  chan error
 }
 
-const minLogCopTaskTime = 50 * time.Millisecond
```

Comment:
A large query may generate many concurrent cop tasks, logging everyone is unnecessary.